### ISSUE TITLE
First time usage hits the ratelimit constantly

### ISSUE THREAD

**Issue by bravecobra** [Memnarch/Delphinus#66]
Upon first usage, the online list is empty. Hitting refresh then hits the ratelimit constantly, making it unusable.

![image](https://user-images.githubusercontent.com/391353/45622870-cb5bae00-ba85-11e8-801e-811e468f589c.png)

**Comment by Memnarch** [comment]
Ahh dang, still haven't corrected the error message to give a clue on how to solve this:
https://github.com/Memnarch/Delphinus/wiki/Common-Errors

Maybe i should give Delphinus a Loginmask so you don't have to do this manually...

**Comment by bravecobra** [comment]
Yes, while you guided me towards fixing the problem, a more descriptive error message would have avoided me raising an issue here. I'd suggest using the opportunity to update the error message with at least the mention of the OAuthToken.

**Comment by Memnarch** [comment]
That's what I'll do
